presentation/src/main/java/com/android254/presentation/common/components/ToggleButtonGroup.kt

```diff
@@ -94,7 +84,14 @@ fun MultiToggleButton(
                         .defaultMinSize(minHeight = buttonHeight)
                         .offset(x = offset),
                     buttonShape = buttonShape,
-                    border = border,
+                    border = BorderStroke(
+                        width = borderSize,
+                        color = if (isButtonSelected) {
+                            ChaiGrey90
+                        } else {
+                            MaterialTheme.chaiColorsPalette.inactiveMultiSelectButtonBorderColor
+                        }
```

Comment:
can't we have this in the chai colors so that here we just use ` MaterialTheme.chaiColorsPalette.inactiveMultiSelectButtonBorderColor` and ` MaterialTheme.chaiColorsPalette.activeMultiSelectButtonBorderColor` if button is selected or not.